src/include/mpir_memkind.h

```diff
@@ -0,0 +1,19 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2018 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+#ifndef MPIR_MEMKIND_H_INCLUDED
+#define MPIR_MEMKIND_H_INCLUDED
+
+#include "mpichconf.h"
+
+typedef enum MPII_Memory_kind {
+    MPIR_MEMKIND__MCDRAM = 0,
+    /* Future memory types must be added after MCDRAM */
+    MPIR_MEMKIND__NUM,
```

Comment:
What is `NUM`?  If you want the number of memory types, why isn't `DRAM` in that list?